docs/UserGuide.md

```diff
@@ -342,3 +315,96 @@ Action | Format, Examples
 **Undo** | `undo`
 **Help** | `help`
 **Exit** | `exit`
+
+## List of acceptable values
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
+.tg .tg-fymr{border-color:inherit;font-weight:bold;text-align:left;vertical-align:top}
+</style>
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-0pky"><span style="font-weight:bold">Command</span></th>
+    <th class="tg-fymr">Field</th>
+    <th class="tg-0pky"><span style="font-weight:bold">Acceptable values</span></th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0pky" rowspan="6"><code>createMember</code> / <code>editMember</code></td>
+    <td class="tg-0pky">MEMBER_NAME</td>
+    <td class="tg-0pky">Only contain alphanumeric characters and spaces, and should not be blank</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">GENDER</td>
+    <td class="tg-0pky"><code>Male</code>, <code>Female</code>, <code>Others</code></td>
```

Comment:
Gender actually takes in mixed case variants as well. I opened #244 to include a card but do you have any ideas to integrate it here in the table aside from adding a card?

Reply:
I think adding an info card will be good here

Reply:
Okay I will proceed with it after this is merged